vars/lvCloseLabVIEW.groovy

```diff
@@ -1,4 +1,6 @@
-def call(lvVersion){
-   def versionPath = env."labviewPath_${lvVersion}"
-   bat "LabVIEWCLI -LabVIEWPath \"${versionPath}\" -OperationName CloseLabVIEW -LogFilePath \"$WORKSPACE\\lvCloseLabVIEW.log\" -LogToConsole false"
+def call() {
+   try {
+      bat "taskkill /IM labview.exe /F"
```

Comment:
Is this ignore simply to account for a case where `labview.exe` is not found or is it to ignore if it can't be killed? In the can't be killed case, this seems no different than the `CloseLabVIEW` operation.

Reply:
The `CloseLabVIEW` operation fails if LabVIEW is in a bad state, i.e. where it is running but the CLI is unable to connect to it. Using `taskkill` ensures that we kill the process.

Unlike the `CloseLabVIEW` operation which will start LabVIEW just to close it, `taskkill` will "fail" if LabVIEW has already been closed. Since all we care about is that LabVIEW _is_ closed, not that `taskkill` was the thing to kill it, ignoring the error seems reasonable.